perceval/backend.py

```diff
@@ -291,7 +291,7 @@ class BackendCommandArgumentParser:
     """
     def __init__(self, from_date=False, to_date=False, offset=False,
                  basic_auth=False, token_auth=False, archive=False,
-                 aliases=None):
+                 aliases=None, categories=[]):
         self._from_date = from_date
         self._to_date = to_date
         self._archive = archive
```

Comment:
I would store the categories in a variable to simplify the tests, something like:
`self._categories = categories`

what do you think ?

Reply:
Agreed @valeriocos, its more organised, I will update, thanks :)